src/common/restext.cpp

```diff
@@ -719,6 +719,8 @@ void InitializeRestext()
     stringsCbot[CBot::CBotErrNoPublic]      = TR("Public required");
     stringsCbot[CBot::CBotErrNoExpression]  = TR("Expression expected after =");
     stringsCbot[CBot::CBotErrAmbiguousCall] = TR("Ambiguous call to overloaded function");
+    stringsCbot[CBot::CBotErrNoPrivilege]   = TR("Access privilege required");
```

Comment:
When editing `restext.cpp`, you should run `make update-pot` to update the translation files. At some point I said I would properly document this as a comment somewhere in the file but I obviously forgot...